modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -5,20 +5,339 @@ endif::[]
 
 This document helps you use Object Store v2 (OSv2) to store and view data.
 
-This tutorial works with Studio 6 using Mule 3.
-
 == Prerequisites
 
-This document assumes you are familiar with Anypoint Studio, using curl from a command line, and the use of Anypoint Platform and Runtime Manager. For more resources, view the See Also section of this document.
+This document assumes you are familiar with Anypoint Studio, using https://curl.haxx.se/[curl] from a command line, 
+and the use of Anypoint Platform and Runtime Manager. Ensure that curl is installed on 
+your computer. If you are using a Mac or Linux, the curl command is built in. For Windows, 
+download and install curl from the net. Alternatively, you can use Postman or browser plugins instead of curl.
+
+You also need to have a Design environment configured in Anypoint Platform > Access Management > Environments. If you don't see Access Management in Anypoint Platform, contact your site 
+administrator.
 
-== To Install the Object Store Connector
+== Install the Object Store Connector
 
 . In Anypoint Studio, click the Exchange icon in the Studio task bar.
 . Click Login in Anypoint Exchange.
 . Search for the "objectstore" connector and click Install.
 . Follow the prompts to install the connector.
 
-== Create a Studio Project
+== Create a Design Center Application
+
+IMPORTANT: Disable password programs such as LastPass or Okta in your browser before using Design Center. Otherwise, these programs inserts passwords in the HTTP Listener’s TLS tab. The passwords in the TLS tab cause your application's deployment to fail.
```

Comment:
Can you check with the flow designer team if this password bug is still present in design center?  If not we can remove this warning.

Reply:
This bug is still in force. The flow designer team may fix it in the future but for documenting now, this must be said or users with LP/Okta deployments will fail every time. You can't just delete the passwords when these services operate -- the apps automatically fill in the TLS passwords as soon as focus changes.